services/QuillLMS/app/views/application/_ortto.html.erb

```diff
@@ -5,10 +5,10 @@
     ap3c.cmd = ap3c.cmd || [];
 
     ap3c.cmd.push(function() {
-        ap3c.init('<%= ENV["ORTTO_KEY"] %>', 'https://capture-api.autopilotapp.com/');
+        ap3c.init('<%= ENV["ORTTO_KEY"] %>', 'https://track.quill.org/');
         ap3c.track({"v":0,"email":"<%= raw(current_user&.email) %>","first":"<%= raw(current_user&.first_name) %>","last":"<%= raw(current_user&.last_name) %>"});
     });
     ap3c.activity = function(act) { ap3c.act = (ap3c.act || []); ap3c.act.push(act); };
-    var s, t; s = document.createElement('script'); s.type = 'text/javascript'; s.src = "https://cdn3l.ink/app.js";
```

Comment:
There is one reference in secure_headers that can probably be removed: https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/config/initializers/secure_headers.rb#L59